idea/src/org/jetbrains/kotlin/idea/inspections/collections/SimplifiableCallChainInspection.kt

```diff
@@ -35,6 +39,10 @@ class SimplifiableCallChainInspection : AbstractCallChainChecker() {
                         }
                     ) return@check false
                 }
+                if (!conversion.enableSuspendFunctionCall && firstResolvedCall.call.callElement.anyDescendantOfType<KtCallExpression> {
```

Comment:
I would do something with the formatting here :)
Maybe, extract the `firstResolvedCall.call.callElement.anyDescendantOfType()` call so the whole `if` condition fit into a single line?